include/envoy/request_info/request_info.h

```diff
@@ -68,66 +68,125 @@ class RequestInfo {
    */
   virtual void onUpstreamHostSelected(Upstream::HostDescriptionConstSharedPtr host) PURE;
 
+  /**
+   * @return the # of body bytes received in the request.
+   */
+  virtual uint64_t bytesReceived() const PURE;
+
+  /**
+   * @return the protocol of the request.
+   */
+  virtual const Optional<Http::Protocol>& protocol() const PURE;
+
+  /**
+   * Set the request's protocol.
```

Comment:
We tend towards individual argument comments so
@param protocol ...
here and for other setter parameters below.

Reply:
Also not sure if this one was missed or if you're still working, so bumping it just in case :-)

Reply:
Missed it (the diff really stirred up all of these)! Just going to fix it for all methods here and be done with it 😅